const main = async () => {
  try {
    // TODO: Refactor seeder to insert the data into the database, returning early for now
    console.error(
      "Refactor seeder to insert the data into the database, returning early for now"
    );
    return;
  } catch (error) {
    console.error(error);
    process.exit(1);
  }
};

main();
